packages/auth/src/openapi/specs/id-provider.yaml

```diff
@@ -39,6 +39,8 @@ paths:
               description: Interaction id
         '401':
           description: Unauthorized
+        '500':
+          description: Internal Server Error
```

Comment:
Would it be good to provide the actual GNAP response error objects in the IDP spec?

Reply:
I'll add them since it's a pretty bare PR as it is right now